tensorflow/python/kernel_tests/embedding_ops_test.py

```diff
@@ -563,7 +563,7 @@ def testHigherRankMaxNorm(self):
           # Compare nonsharded to gather
           simple = embedding_ops.embedding_lookup(
               params, ids, max_norm=1.0).eval()
-          self.assertAllEqual(simple, array_ops.gather(params_norm, ids).eval())
+          self.assertAllClose(simple, array_ops.gather(params_norm, ids).eval())
```

Comment:
It would be great to put the comments alongside the change in the code. Same for the change below.
Thanks for the catch.

Reply:
@protoget Thanks for the review.  I've added an explanatory comment to each of the changes.